vendor/plist_to_html/Makefile

```diff
@@ -19,12 +21,38 @@ package: dep
 	cp plist_to_html/dist/js/bugviewer.js $(DIST_DIR)
 	cp plist_to_html/PlistToHtml.py $(BIN_DIR)/plist-to-html
 
-dep:
+dist_dir:
 	mkdir -p $(DIST_DIR)
-	curl -sSfLk --get $(CODEMIRROR)/codemirror.min.js -z $(DIST_DIR)/codemirror.min.js -o $(DIST_DIR)/codemirror.min.js
-	curl -sSfLk --get $(CODEMIRROR)/codemirror.min.css -z $(DIST_DIR)/codemirror.min.css -o $(DIST_DIR)/codemirror.min.css
-	curl -sSfLk --get $(CODEMIRROR)/mode/clike/clike.min.js -z $(DIST_DIR)/clike.min.js -o $(DIST_DIR)/clike.min.js
-	curl -sSfLk --get https://raw.githubusercontent.com/codemirror/CodeMirror/master/LICENSE -z $(DIST_DIR)/codemirror.LICENSE -o $(DIST_DIR)/codemirror.LICENSE
+
+codemirror: $(DIST_DIR)/codemirror.min.js $(DIST_DIR)/codemirror.min.css $(DIST_DIR)/codemirror.LICENSE
+
+$(DIST_DIR)/codemirror.min.js: dist_dir
+	[ -f $(DIST_DIR)/codemirror.min.js ] && : || \
+	curl -sSfLk --get $(CODEMIRROR)/codemirror.min.js \
+			 -z $(DIST_DIR)/codemirror.min.js \
+			 -o $(DIST_DIR)/codemirror.min.js
+
+$(DIST_DIR)/codemirror.min.css: dist_dir
+	[ -f $(DIST_DIR)/codemirror.min.css ] && : || \
+	curl -sSfLk --get $(CODEMIRROR)/codemirror.min.css \
+			 -z $(DIST_DIR)/codemirror.min.css \
+			 -o $(DIST_DIR)/codemirror.min.css
+
+$(DIST_DIR)/codemirror.LICENSE: dist_dir
+	[ -f $(DIST_DIR)/codemirror.LICENSE ] && : || \
+	curl -sSfLk --get https://raw.githubusercontent.com/codemirror/CodeMirror/master/LICENSE \
+			 -z $(DIST_DIR)/codemirror.LICENSE \
+			 -o $(DIST_DIR)/codemirror.LICENSE
+
+clike: $(DIST_DIR)/clike.min.js
```

Comment:
Why did you create a separate target for this? `clike` is part of the CodeMirror library. It looks to me that the plist-to-html plugin depends on two libraries: codemirror and clike, but clike is not a library.

I think it would be better to use the `codemirror` target instead of creating a new one.

Reply:
The fact that it was under a `mode` subfolder under CodeMirror's host must have mislead me when I made this patch.